File: solution/java/findMaxConsecutiveOnes.java
public class findMaxConsecutiveOnes {
    public static void main(String[] args) {
        
    }


     public int findMaxConsecutiveOnes(int[] nums) {
        int res = 0;

        for(int lo = 0, hi = 0;  hi < nums.length;hi++) {
            if(nums[hi] != 1){
                res = Math.max(res, hi  - lo);
            }

            if(hi == nums.length - 1){
                res = Math.max(res, hi - lo + 1);
            }
        }

        return res;
    }

}

File: solution/py/findMaxConsecutiveOnes.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        n = len(nums)
        res = 0
        lo = 0
        for  hi in range(len(nums)):
            if nums[hi] == 1:
                res = max(res, hi - lo + 1)  
            else: 
                lo = hi + 1

    
        return res
        